Author: Evgeni Golov <sargentd@die-welt.net>
Description: include thinkpad_ec.h directly, not via linux/thinkpad_ec.h
--- tp-smapi.orig/hdaps.c
+++ tp-smapi/hdaps.c
@@ -34,7 +34,7 @@
 #include <linux/timer.h>
 #include <linux/dmi.h>
 #include <linux/jiffies.h>
-#include <linux/thinkpad_ec.h>
+#include "thinkpad_ec.h"
 #include <linux/pci_ids.h>
 #include <linux/version.h>
 
--- tp-smapi.orig/thinkpad_ec.c
+++ tp-smapi/thinkpad_ec.c
@@ -34,7 +34,7 @@
 #include <linux/dmi.h>
 #include <linux/ioport.h>
 #include <linux/delay.h>
-#include <linux/thinkpad_ec.h>
+#include "thinkpad_ec.h"
 #include <linux/jiffies.h>
 #include <asm/io.h>
 
--- tp-smapi.orig/tp_smapi.c
+++ tp-smapi/tp_smapi.c
@@ -42,7 +42,7 @@
 #include <linux/mc146818rtc.h>	/* CMOS defines */
 #include <linux/delay.h>
 #include <linux/version.h>
-#include <linux/thinkpad_ec.h>
+#include "thinkpad_ec.h"
 #include <linux/platform_device.h>
 #include <asm/uaccess.h>
 #include <asm/io.h>
